tools/tinygobb/README.md

```diff
@@ -0,0 +1,186 @@
+# Status of tinygo in u-root
+
+Currently, many of the busybox commands fail to build using tinygo.
+This document aims to cover the current process of enabling all subcommands to be built using tinygo.
+While enabling more and more commands the list of working commands in this document will be updated.
+
+## building
+Since currently the tinygo support is lacking, many commands got the `!tinygo` build tag.
+This prohibits building, so anyone recreating the build steps should be aware that this has to be removed for each command.
+The list below is the result of building each subcommand individually for x86_64 linux.
```

Comment:
Is it limited to x86-64 ? we need them working on arm64 too

Reply:
It is probably not limited to x86_64, but the current tests were executed on x86_64.
Maybe a matrix would be better to display this information.

Reply:
I have found that if it works on x86_64, it also works on risc-v and arm.

Reply:
Added a tracker item to add compatibility matrix.  https://github.com/orgs/u-root/projects/2/views/1?pane=issue&itemId=66662510.